apps/govern/common-util/config/wagmi.ts

```diff
@@ -27,30 +27,27 @@ export const SUPPORTED_CHAINS: [Chain, ...Chain[]] = [
   ...(process.env.NEXT_PUBLIC_IS_CONNECTED_TO_LOCAL === 'true' ? [hardhat] : []),
 ];
 
+const SUPPORTED_CHAINS_WITH_RPCS = SUPPORTED_CHAINS.map((chain) => {
+  const defaultRpc = RPC_URLS[chain.id] || chain.rpcUrls.default.http[0];
+  return {
+    ...chain,
+    rpcUrls: {
+      ...chain.rpcUrls,
+      default: { http: [defaultRpc] },
+    },
+  } as Chain;
+});
+
 const walletConnectMetadata = {
   name: 'OLAS Govern',
   description: 'OLAS Govern Web3 Modal',
   url: 'https://govern.olas.network',
   icons: ['https://avatars.githubusercontent.com/u/37784886'],
 };
 
-export const wagmiConfig = createConfig({
-  chains: SUPPORTED_CHAINS,
-  connectors: [
-    injected(),
-    walletConnect({
-      projectId: process.env.NEXT_PUBLIC_WALLET_PROJECT_ID || '',
-      metadata: walletConnectMetadata,
-      showQrModal: false,
-    }),
-    safe(),
-    coinbaseWallet({
-      appName: walletConnectMetadata.name,
-    }),
-  ],
-  transports: SUPPORTED_CHAINS.reduce(
-    // TODO: merge two sources of RPCs into one?
-    (acc, chain) => Object.assign(acc, { [chain.id]: http(RPC_URLS[chain.id]) }),
-    {},
-  ),
+export const wagmiConfig = defaultWagmiConfig({
```

Comment:
had to use `defaultWagmiConfig` because it seems we should define safe connector somehow differently. works in registry, so just copied from there